services/core/shared/core/accountUtils.js

```diff
@@ -33,7 +33,8 @@ const isStringType = value => typeof value === 'string';
 
 const parseAddress = address => isStringType(address) ? address.toUpperCase() : '';
 
-const validateAddress = address => isStringType(address) && address.match(/^[0-9]{1,20}[L|l]$/g);
+const validateAddress = address => isStringType(address)
+    && (address.match(/^[0-9]{1,20}[L|l]$/g) || address.match(/^[a-z0-9]{40}$/g));
```

Comment:
This regex seems too liberal. As per `Lisk Service SDK v5 Update Specification` ideally the regex must be: `/^lsk([a-hjkm-z]|[2-9]){38}$/`
```suggestion
    && (address.match(/^[0-9]{1,20}[L|l]$/g) || address.match(/^lsk([a-hjkm-z]|[2-9]){38}$/g));
```

Reply:
The reason behind is we are using binary encoded addresses to retrieve data from `lisk-core`. So, address looks like: `3b3e137b1bec6f20c9a8b2ad4f5784661fb0fa79`.